cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -24,7 +28,7 @@ def map_tags(appliance, provider, request):
 
 
 @pytest.fixture(scope='function')
-def vm(appliance, provider):
+def tagged_vm(appliance, provider):
     # cu-24x7 vm is tagged with test:testing in provider
```

Comment:
In this fixture, ```appliance``` is [accessed from ```provider``` object.](https://github.com/ManageIQ/integration_tests/pull/8543/files#diff-dcc49450aead0469ee3a75df5dbbf539R34)
So according to me, we can remove ```appliance``` argument.